test/controllers/user_controller_test.exs

```diff
@@ -115,4 +115,55 @@ defmodule CodeCorps.UserControllerTest do
     assert response(conn, 204)
     refute Repo.get(User, user.id)
   end
+
+  describe "email_available" do
+    test "returns { available: true, valid: true } when email is valid and available", %{conn: conn} do
```

Comment:
The `returns { available: true, valid: true }` seems to do extra work of describing internals of the test, instead of just generalizing its conditions.